recipes/codetiming/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "codetiming" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/codetiming-{{ version }}.tar.gz
+  sha256: 7f9949dad6438f458a22b811b0738c55e45f685d27551352871e15811f3a5e89
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - flit-core
+    - python >=3.6
+  run:
+    - python >=3.6
```

Comment:
Python 3.6 requires `dataclasses`. You should add it here. It will be empty for python >=3.7.
See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=dataclasses#empty-python-packages